src/coreclr/inc/dacprivate.h

```diff
@@ -610,6 +610,8 @@ struct MSLAYOUT DacpTieredVersionData
         OptimizationTier_OptimizedTier1,
         OptimizationTier_ReadyToRun,
         OptimizationTier_OptimizedTier1OSR,
+        OptimizationTier_InstrumentedTier,
```

Comment:
I kind of think this should be `InstrumentedTier0` (though we don't use Tier0 here -- so arguably `InstrumentedQuickJitted`). Otherwise, I have to keep reminding myself that this is not optimized code.

Likewise below I'd use `InstrumentedTier1Optimized`.

Reply:
Good idea, renamed to match existing conventions